e2e-tests/src/tests/studies/ourhealth/new-registration-age18.test.ts

```diff
@@ -66,18 +69,18 @@ test.describe('Home page', () => {
       const consent = new StudyConsent(page)
 
       let progress = await consent.progress()
-      expect(progress).toStrictEqual('Page 1 of 3')
+      await expect(progress).toHaveText('Page 1 of 3')
       await consent.clickByRole('button', 'Next')
 
       progress = await consent.progress()
-      expect(progress).toStrictEqual('Page 2 of 3')
+      await expect(progress).toHaveText('Page 2 of 3')
       await consent.agree()
       await consent.clickByRole('button', 'Next')
 
 
       progress = await consent.progress()
-      expect(progress).toStrictEqual('Page 3 of 3')
-      await consent.fillIn(data.QLabel.FullName, 'Tony Junior Stark', page.locator('body')) // TODO randomize
+      await expect(progress).toHaveText('Page 3 of 3')
+      await page.locator('body').getByLabel('Full legal name *').fill('Tony Junior Stark')
```

Comment:
This could have been accomplished with a `new Question` -> `question.fill` but imo I think we should refactor all of our tests to just do something like this. I think the nested interfaces we have made me spend at least an hour longer than I needed to to refactor these...